elements/pfe-navigation/src/_pfe-navigation--grid.scss

```diff
@@ -0,0 +1,106 @@
+.pfe-navigation__dropdown--12-column-grid {
+
+  // Fallback flexbox dropdown layout
+  // --------------------------------------------------------
+  display: flex;
+  flex-wrap: wrap;
+  // width: 100%;
+  margin-left: #{$gutter / -2};
+  margin-right: #{$gutter / -2};
+  gap: 0;
+
+  @for $i from 1 to 13 {
+    .col-xs-#{$i} {
+      $col-width: #{$i / 12 * 100%};
+      flex: 0 0 calc(#{$col-width} - #{$gutter / 2});
+      width: $col-width;
+      max-width: $col-width;
+      margin: 0 0 $gutter;
+      padding-left: #{$gutter / 2};
+      padding-right: #{$gutter / 2};
+    }
+  }
+
+  @each $breakpoint, $breakpoint-value in $breakpoints {
+    @if $breakpoint != 'secondary-links-expand' and $breakpoint != 'nav-expand' {
+      @for $i from 1 to 13 {
+        .col-#{$breakpoint}-#{$i} {
+          @include breakpoint($breakpoint) {
+            $col-width: #{$i / 12 * 100%};
+            flex: 0 0 calc(#{$col-width} - #{$gutter / 2});
+            width: $col-width;
+            max-width: $col-width;
+            margin: 0 0 $gutter;
+            padding-left: #{$gutter / 2};
+            padding-right: #{$gutter / 2};
+          }
+        }
+      }
+    }
+  }
+
+  .pfe-navigation__footer {
+    width: 100%;
+    padding-left: #{$gutter / 2};
+    padding-right: #{$gutter / 2};
+  }
+
+  // CSS Grid dropdown layout
+  // --------------------------------------------------------
+  @supports (display: grid) {
+    display: grid;
+    grid-template-columns: repeat(12, minmax(0, 1fr));
+    gap: $gutter;
+    padding-left: 0;
+    padding-right: auto;
+
+    @for $i from 1 to 13 {
+      .col-xs-#{$i} {
```

Comment:
This seems a bit duplicative of the grid styles in pfe-styles.  Why not add to that asset instead of build something new and separate that developers have to learn?

| Class | Description |
| -------------- | ----------- |
| `pfe-l-grid` | Base grid class *required* |
| `pfe-l-grid-fill-height` | Allows children elements to fill their container's height completely; equivalent of flex-grow |
| `pfe-m-gutters` | Adds gutters based on `--pfe-theme--container-spacer` |
| `pfe-m-all-*[1-12]*-col` | Sets width of children in grid to *[1-12]* columns |
| `pfe-m-all-*[1-12]*-col-on-[xs-xl]` | Sets width of children to *[1-12]* columns on specified breakpoint *[xs-xl]* |